src/app/users/editServices.js

```diff
@@ -6,6 +6,17 @@ const { getApplication } = require('./../../infrastructure/applications');
 const PolicyEngine = require('login.dfe.policy-engine');
 const policyEngine = new PolicyEngine(config);
 
+const renderEditServicePage = (req, res, model) => {
+  //TODO: For now using currentPage as "users" but need to change to "services"
+  const isSelfManage = true; //isSelfManagement(req);
```

Comment:
you'll need to remove the hardcoded "true" I suppose

Reply:
I don't think this change will be enough to keep the old "edit a service" journey within "manage users" using old UI in the scenario when the approver is editing its own services

Reply:
Done!